### PR TITLE
docs: update documentation under tutorials

### PR DESCRIPTION
**Pull request by cvaldez98**
#### Description of Change
Change the documentation to reflect the fact that Electron is now kept up to date with Chromium.

#### Checklist

- [x] relevant documentation is changed or added

#### Release Notes
notes: Update documentation to reflect that Electron is now up to date with a stable version of Chromium.

### CODE REVIEW

File: docs/tutorial/security.md [hunk 1]
```diff
@@ -27,17 +27,11 @@ see [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md)
...
-hand-editing dozens or even hundreds of files. Given the resources and
-contributions available today, Electron will often not be on the very latest
-version of Chromium, lagging behind by several weeks or a few months.
-
-We feel that our current system of updating the Chromium component strikes an
-appropriate balance between the resources we have available and the needs of
-the majority of applications built on top of the framework. We definitely are
-interested in hearing more about specific use cases from the people that build
-things on top of Electron. Pull requests and contributions supporting this
+Electron is now up to date with the latest chromium release. For more information,
```
Comment (malept) [inline docs/tutorial/security.md]:
```suggestion
Electron keeps up to date with the latest stable Chromium release. For more information,
```

File: docs/tutorial/security.md [hunk 2]
```diff
@@ -27,17 +27,11 @@ see [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md)
...
+see the [Electron blog](https://electronjs.org/blog/12-week-cadence).
+
+We are interested in hearing more about specific use cases from the people that 
+build things on top of Electron. Pull requests and contributions supporting this
 effort are always very welcome.
```
Comment (malept) [inline docs/tutorial/security.md]: Is this paragraph necessary anymore?

**Comment by malept** [comment]
CC: @electron/wg-docs-tools

File: docs/tutorial/security.md (hunk 2 above)
Comment (cvaldez98) [reply docs/tutorial/security.md]: We can ✂️it, sure!

File: docs/tutorial/security.md [hunk 3]
```diff
@@ -27,18 +27,8 @@ see [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md)
...
-contributions available today, Electron will often not be on the very latest
-version of Chromium, lagging behind by several weeks or a few months.
-
-We feel that our current system of updating the Chromium component strikes an
-appropriate balance between the resources we have available and the needs of
-the majority of applications built on top of the framework. We definitely are
-interested in hearing more about specific use cases from the people that build
-things on top of Electron. Pull requests and contributions supporting this
-effort are always very welcome.
+Electron keeps up to date with the latest stable Chromium release. For more information,
```
Comment (MarshallOfSound) [inline docs/tutorial/security.md]:
```suggestion
Electron keeps up to date with alternating Chromium releases. For more information,
```

File: docs/tutorial/security.md [hunk 4]
```diff
@@ -27,18 +27,8 @@ see [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md)
...
+see the [Electron blog](https://electronjs.org/blog/12-week-cadence).
```
Comment (MarshallOfSound) [inline docs/tutorial/security.md]:
```suggestion
see the [Electron Release Cadence blog post](https://electronjs.org/blog/12-week-cadence).
```

File: docs/tutorial/security.md (hunk 3 above)
Comment (malept) [reply docs/tutorial/security.md]: I should have probably reread the blog post before suggesting changes 😅

**Pull request merged**